sherpa/astro/datastack/tests/test_datastack.py

```diff
@@ -680,7 +680,7 @@ def test_case_7(self):
 
         assert f == [1, 2]
 
-        f = datastack.query_by_obsid('7867')
+        f = datastack.query_by_obsid(7867)
```

Comment:
I should point out that I'm jet-lagged, so about the only thing I feel up to is nit-picking [1] in a code review ;-)

[1] not sure of the equivalent Italian expression